backend/taggle-web/src/test/java/kr/taggle/common/config/EtagConfigTest.java

```diff
@@ -0,0 +1,86 @@
+package kr.taggle.common.config;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.filter.ShallowEtagHeaderFilter;
+
+@SpringBootTest
+public class EtagConfigTest {
+
+    private MockMvc mockMvc;
+
+    @BeforeEach
+    void setUp(final WebApplicationContext webApplicationContext) {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext)
+                .addFilter(new ShallowEtagHeaderFilter())
+                .build();
+    }
```

Comment:
```suggestion
@SpringBootTest
@AutoConfigureMockMvc
public class EtagConfigTest {

    @Autowired
    private MockMvc mockMvc;
```
`@AutoConfigureMockMvc`를 사용하면 MockMVC에 대한 기본 설정을 해줍니다. 물론 [Filter](https://github.com/spring-projects/spring-boot/blob/v1.4.0.RELEASE/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/web/servlet/SpringBootMockMvcBuilderCustomizer.java#L83-L94)에 관한 것도요. 테스트도 정상 동작합니다.😊😊
이건 [참고자료](https://stackoverflow.com/questions/38746850/how-can-i-use-webmvctest-and-also-add-in-my-own-custom-filters)입니다.
하지만 단점으로 커스텀하기 어렵습니다. 그래서 보시고 써도 괜찮을 것 같다면 사용하셔도 될 것 같습니다.😊😊

Reply:
오... 리뷰하다가 새로운 애노테이션 하나 배워가네요. 감사합니다. 티거!

Reply:
@AutuConfigureMockMvc 사용해보고 싶은데 커스텀이 어렵다는게 걱정이되는거 같아요!

그래도 ! 일단 진행해보겠습니다!

너무 시간이 소요되는거같은 생각이들면 일단 저 상태로 진행 하도록 할께요!